jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -140,9 +140,23 @@ public void assignGroupToRole(final String serverTemplateId, final String contai
     }
 
     @Override
-    public void assignGroupAndUserToRole(final String serverTemplateId, final String containerId, final String caseId, final String roleName, final String user, final String group) {
-        client.assignGroupToRole(containerId, caseId, roleName, group);
-        client.assignUserToRole(containerId, caseId, roleName, user);
+    public void assignToRole(final String serverTemplateId, final String containerId, final String caseId,
+                             final String roleName, final List<String> usersAssignments, final List<String> groupsAssignments,
+                             final List<String> oldUserAssignments, final List<String> oldGroupsAssignments) {
+
+        List<String> usersToRemove = new ArrayList<String>(oldUserAssignments);
```

Comment:
Suggestion to move this bit into the presenter and use only the remove/assign methods.